### PR TITLE
keep it alive

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### CODE REVIEW

File: media/system/js/keepalive.js [hunk 1]
```diff
@@ -11,37 +11,26 @@
...
-				{
-					// Do nothing
-				}
-			});
-		}, keepaliveInterval);
-
-	});
-
-})(window, document, Joomla);
+((window, document, getOptions, Request) => {
```
Comment (dneukirchen) [inline media/system/js/keepalive.js]: filename.es6.js ?

**Comment by dneukirchen** [comment]
please use .es6.js suffix for es6 files

**Comment by dgrammatiko** [comment]
can we change the es6.js to src.js?

File: media/system/js/keepalive.es6.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * Keepalive javascript behavior
+ *
+ * Used for keeping the session alive
+ *
+ * @package  Joomla
+ * @since    3.7.0
+ */
+((window, document, getOptions, Request) => {
```
Comment (dneukirchen) [inline media/system/js/keepalive.es6.js]: Why is Request uppercased?

**Comment by dneukirchen** [comment]
please add the compiled file

**Comment by anuragteapot** [comment]
@dgrammatiko can you please make this PR to get it merge .

**Pull request closed without merging**